src/types/mod.rs

```diff
@@ -169,6 +169,21 @@ impl Flag<'static> {
     }
 }
 
+impl<'a> fmt::Display for Flag<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match *self {
+            Flag::Seen => write!(f, "{}", "\\Seen"),
+            Flag::Answered => write!(f, "{}", "\\Answered"),
+            Flag::Flagged => write!(f, "{}", "\\Flagged"),
+            Flag::Deleted => write!(f, "{}", "\\Deleted"),
+            Flag::Draft => write!(f, "{}", "\\Draft"),
+            Flag::Recent => write!(f, "{}", "\\Recent"),
+            Flag::MayCreate => write!(f, "{}", "\\*"),
```

Comment:
Is this really the right flag for `MayCreate`?!

Reply:
Yup. See PERMANENTFLAGS section here: https://tools.ietf.org/html/rfc3501#section-7.1

The `system` fn implementation for `Flag` here also reads this string as that flag:  https://github.com/jonhoo/rust-imap/blob/master/src/types/mod.rs#L166

I was also a bit surprised. :-)